src/main/java/frc/robot/RobotContainer.java

```diff
@@ -136,6 +141,14 @@ private void configureBindings() {
                 })
             .ignoringDisable(true));
 
+    IS_DISABLED
+        .and(() -> RobotController.getBatteryVoltage() >= 12)
+        .onTrue(led.setPattern(LEDPattern.solid(Color.kGreen)));
```

Comment:
I think this will cause extra work by setting the pattern even if the color is not changing.

It will work - maybe a future improvement for LED Subsystem.  Have an enum with the state symbols